src/Pages/Contact.tsx

```diff
@@ -85,7 +86,7 @@ export default function Contact() {
     return (
         <Content>
             <TopPageSpacer />
-            <div data-aos="fade-up">

```

Comment:
Ukryłem kafelki koordynatorów i zmiłem rozmiar czarnej sekcji pod nią, aby nie kuła w oczy, po otrzymaniu zdjęć, trzeba to włączyć i zmienić rozmiay w dwóch powyższych stylach